channel-api/src/main/java/com/synopsys/integration/alert/channel/api/DistributionEventReceiver.java

```diff
@@ -43,6 +45,9 @@ public final void handleEvent(DistributionEvent event) {
         Optional<D> details = jobDetailsAccessor.retrieveDetails(event.getJobId());
         if (details.isPresent()) {
             try {
+                if (logger.isDebugEnabled()) {
```

Comment:
These if's use the normal logger throughout the PR so that we only log these messages when the User has enabled debug logging for all of Alert.